move_tab.py

```diff
@@ -12,51 +12,27 @@
 
 http://github.com/FMCorz/MoveTab
 """
+import sublime_plugin
 
-import sublime, sublime_plugin
 
 class MoveTabCommand(sublime_plugin.WindowCommand):
 
-	def run(self, position):
-		position = str(position)
-		view = self.window.active_view()
-		(group, index) = self.window.get_view_index(view)
-		if index < 0:
-			return
-		count = len(self.window.views_in_group(group))
-
-		direction = None
-		if position.startswith('-') or position.startswith('+'):
-			direction = position[0]
-			steps = int(position[1:])
-			if direction == '-':
-				position = index - steps
-			else:
-				position = index + steps
-
-		position = int(position)
-		if position < 0:
-			position = count - 1
-		elif position > count - 1:
-			if direction: position = 0
-			else: position = count - 1
-
-		# Avoid flashing tab when moving to same index
-		if position == index:
-			return
-
-		self.window.set_view_index(view, group, position)
-		self.window.focus_view(view)
-
-	def is_enabled(self):
-		view = self.window.active_view()
-		if view == None:
-			return False
-		(group, index) = self.window.get_view_index(view)
-		return len(self.window.views_in_group(group)) > 1
-
-	def is_visible(self):
-		return True
-
-	def description(self):
-		return None
+    def run(self, position):
+        view = self.window.active_view()
+        group, index = self.window.get_view_index(view)
+        if index < 0:
+            return
+        count = len(self.window.views_in_group(group))
+
+        if isinstance(position, str) and position[0] in '-+':
+            position = (index + int(position)) % count
+        else:
+            position = min(count - 1, max(0, int(position)))
```

Comment:
```suggestion
        position = (index + int(position)) % count
```

Doesn't this single line do the same?

Reply:
@FichteFoll If there are no signs, then the original one would perform no modular behavior, which means they aren’t the same.

Reply:
It would still clamp the value between `0` and `count - 1`, but you are indeed right that they are *not* the same, because the second case does not use `index`.